### ISSUE TITLE
Maintainers wanted

### ISSUE THREAD

**Issue by nicholastmosher** [jethrogb/rust-core_io#15]
Hi there, I'm looking for a no_std io solution and this project seems to be the best one out there that I've seen. It looks like it hasn't been touched in a year though, and I was wondering whether this isn't maintained or if I don't understand the usage properly. When including this as a dependency `core_io = { version = "0.1", features = [ "alloc", "collections" ] }`, I get the version `0.1.20180307`. If I manually set my compiler to the matching nightly (`rustup override set nightly-2018-03-07`), then the compilation works, but otherwise it doesn't (I would like to use a recent nightly). I'm trying to figure out whether there is something I'm missing here. Are users supposed to "roll their own" core_io based on the compiler version they're using? Or was it intended for this crate to be published on a regular basis?

I managed to make a new patch for `2aa4c46cfdd726e97360c2734835aa3515e8c858` (the 1.33.0 stable release commit) that works, but it's unclear to me how I would go about including this in my project. Any advice would be appreciated!

**Comment by jethrogb** [comment]
Yes it is intended that this crate be updated for every nightly. My own use for this crate has gone away, so maintenance has been... spotty.

There are instructions on how to add a version at https://github.com/jethrogb/rust-core_io#adding-new-nightly-versions, but note that I only accept PRs that add *all* missing versions since the last update.

**Comment by parasyte** [comment]
I was able to get everything working, at last! #18 updates all nightlies up to 2019-04-27

**Comment by jethrogb** [comment]
Published a new version today, thanks so much for your work @parasyte. If you or @nicholastmosher have an ongoing use for this crate, would you consider co-maintaining this with me?

**Comment by parasyte** [comment]
I haven’t used this yet, but I intend to. I’m making a set of Nintendo 64 crates and `core_io` will be useful for FAT32 access (no_std). I would be interested in helping with maintenance as I start to use it.

**Comment by nicholastmosher** [comment]
Hey there,

Thanks @parasyte for putting together an update, I'm planning on trying that out today. It's kind of hard for me to grok what the process was for (successfully) updating the crate, I still don't totally understand my way around the scripts in here. I feel like that's something I would need to more thoroughly understand what to do in order to help maintain this. My use of this crate is just for a pet project, so sometimes I don't get to spend as much time on it as I would like, but I wouldn't mind opening a PR to bump this to a new nightly every once in awhile. I feel like I was on the right track with #16, but I might need a nudge in the right direction if I try again.

**Comment by NickeZ** [comment]
Would it be possible to use `core_io` with a stable compiler?

**Comment by vinc** [comment]
Hello, I couldn't quite grok how to update this crate, it's been documented already but it'd be really cool to have just one script to call, commit, and do a PR <3

I'd like to use https://github.com/rafalh/rust-fatfs for a toy OS I started writing but I'm using the latest nightly and not the one from six months ago..

**Comment by MabezDev** [comment]
@vinc I've updated nightlies up to dec 2019 in #30, you can use my fork until its merged.

**Comment by jethrogb** [comment]
@MabezDev do you want to be added as a crate owner?

**Comment by MabezDev** [comment]
@jethrogb Sure, I'd be happy to help maintain this.

**Comment by MabezDev** [comment]
@jethrogb Would it be possible to get that invite?

**Comment by lexxvir** [comment]
Hi @jethrogb !
Thank you for the crate, it super useful for me!

Could you please add @MabezDev as co-maintainer?
His fork contains patches up to 18.03.2020, so it will be great to release them as a new version on the crates.io.
